avm/bicepconfig.json

```diff
@@ -0,0 +1,14 @@
+// This is a Bicep configuration file. It can be used to control how Bicep operates.
```

Comment:
```suggestion
// This is a Bicep configuration file. It can be used to control how Bicep operates and to customize validation settings for the Bicep linter. The linter uses these settings when evaluating your Bicep files for best practices.
```